gpuarray/include/gpuarray/buffer_blas.h

```diff
@@ -1,12 +1,12 @@
-#ifndef GPUARRAY_BUFFER_BLAS_H
-#define GPUARRAY_BUFFER_BLAS_H
+#ifndef LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
+#define LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
```

Comment:
Yes, a header guard is basically mandatory.  I'm just arguing about the tag itself.